helmfile/charts/rclone-sync/templates/cronjob.yaml

```diff
@@ -0,0 +1,45 @@
+{{- range .Values.buckets }}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ $.Chart.Name }}-{{ .source }}
+  labels:
+    app: {{ $.Chart.Name }}-{{ .source }}
+spec:
+  schedule: {{ .schedule }}
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            app: {{ $.Chart.Name }}-{{ .source }}
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: {{ $.Chart.Name }}
+              image: {{ $.Values.image.repository }}:{{ $.Values.image.tag }}
+              command: ["rclone"]
+              args:
+                - sync
+                - {{ $.Values.config.source.name }}:{{ .source }}
+                {{- if hasKey . "destination" }}
+                - {{ $.Values.config.destination.name }}:{{ .destination }}
+                {{- else }}
+                - {{ $.Values.config.destination.name }}:{{ .source }}
+                {{- end }}
+                - --log-level
+                - DEBUG
+                {{- if $.Values.config.dryrun }}
+                - --dry-run
+                {{- end }}
+              resources: {{ toYaml $.Values.resources | nindent 16 }}
+              volumeMounts:
+                - name: {{ $.Chart.Name }}-config
+                  mountPath: /root/.config/rclone/
```

Comment:
Since this is our image. I think it would look better if we updated the image to not use the root user. If it already runs as non-root, then I think we should put this somewhere else (like `/tmp/rclone` or something) and point to it.
Then we could also add the `CMD ["rclone"]` so that you could skip the `command: ["rclone"]` line

Reply:
Definitely

Reply:
I've added an updated Dockerfile.
Can someone with access to our Docker hub repos build the image, tag it with version `1.2.0` and push it to the `elastisys/rclone-sync` repo?

Reply:
There is now a `1.2.0` version pushed